style.css

```diff
@@ -0,0 +1,61 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    background-color: rgb(191,239,255)
+  }
+  .topnav {
+    background-color: rgb(74, 103, 136);
+    overflow: hidden;
+    margin: 0 auto;
+    text-align: center;
+    
+    padding: 30px;
+    
+  
```

Comment:
I think there shouldn't be emty space for your code seems better looks and doesn't take linters errors.